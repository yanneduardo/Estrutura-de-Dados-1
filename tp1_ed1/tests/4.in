5 
0 0 0 
0 1 2 
0 2 0
0 3 3
0 4 6
1 0 2 
1 1 0
1 2 4 
1 3 3
1 4 0
2 0 0
2 1 4
2 2 0
2 3 7
2 4 3
3 0 3 
3 1 3
3 2 7
3 3 0
3 4 3
4 0 6
4 1 0
4 2 3
4 3 3
4 4 0
