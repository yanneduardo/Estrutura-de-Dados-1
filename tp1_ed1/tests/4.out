0 1 2 4 3 0 
15
